docs/operator-notes.md

```diff
@@ -62,6 +62,14 @@ If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the exi
 
 When resolving paths, Ignition follows symlinks on all but the last element of a path. This ensures existing symlinks on a filesystem can be overwritten while still following symlinks as expected. When writing files, links, or directories, Ignition does not allow following symlinks outside the specified filesystem. When writing files, links, or directories on the `root` filesystem, Ignition follows symlinks as if it were executing in that root; a symlink to `/etc` is followed to `/etc` on the `root` filesystem. When writing files, links, or directories to any other filesystem, Ignition fails if it tries to follow a symlink outside that filesystem.
 
+## Making changes to `/proc`, `/sys`, `/dev`, `/tmp` or `/run`
+
+To create files, directories or symlinks in `/proc`, `/sys` or `/dev`, you should use [sysctl.d config files](https://www.mankier.com/5/sysctl.d) or [udev rules](https://www.mankier.com/7/udev).
+
+Similarly, to make changes under the `/tmp` or `/run` paths, you should use [tmpfiles.d config files](https://www.mankier.com/5/tmpfiles.d).
+
+Those directories are managed by system services (systemd, udev, etc.). Changes under those directories should come from the services' configuration. Otherwise, those services will not be aware of those changes, and may ignore or overwrite them.
```

Comment:
It's even more fundamental than that: those directories on disk are simply mountpoints. At the time Ignition runs, none of those mounts are present, so writing to them will literally write to the disk, which will just be shadowed by the real mounts after switchroot.

Reply:
First, I would like to apologize to @dlipovetsky for the back and forth here as it comes from my initial misunderstanding.

This is what @dlipovetsky had written in a previous version of this PR before I asked to change it: [`b6419e3` (#1697)](https://github.com/coreos/ignition/pull/1697/commits/b6419e33ca939a0d2f3d7712a1e7afc35689a2b2)

If I'm not mistaken, we don't mandate `/tmp` to be a mount point even though it's the most common case right now. But agree that all the other ones are expected to be mount points.

Reply:
I really appreciate your words, @travier!